### PR TITLE
Joe security detonate file playbook fixes

### PR DESCRIPTION
**Pull request by karinafishman**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-31394

## Description
Fixes for the following issues:
- The 'Is the file type supported?' Task in the "Detonate File - JoeSecurity V2" Playbook is not using the correct input values (it should be inputs.File.Type, but it is using 'File.Type'.
- The Description for the Timeout in the "Detonate File - JoeSecurity V2" Playbook is wrong (it says in minutes, but the actual timeout is in Seconds, since it transitioned to the polling command).
- The "Detonate File - Generic" Playbook is giving a Timeout of "15", likely because the input says it's in minutes, but this actually means, that the "Detonate File - JoeSecurity V2" Playbook will fail after 15 seconds.
- The Regex of the 'Is the file type supported?' Task, doesn't work with Word files, since the type doesn't say DOCX or DOC, but "Microsoft Word 2007+"

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SecneurXAnalysis/Playbooks/Detonate_File_-_SecneurX_Analysis.yml [hunk 1]
```diff
@@ -410,7 +410,8 @@ tasks:
         - operator: match
           left:
             value:
-              simple: File.Extension
+              simple: inputs.File.Extension
+            iscontext: true
           right:
             value:
               simple: .*(?:SH|JS|7Z|PY|DOC|RTF|XLS|PPT|PPS|PS1|BAT|EML|EXE|ISO|JAR|TXT|ODT|ODP|ODS|SWF|MSG|MSI|URL|PDF|RAR|VBS|ZIP|CAB|INK|XML|DLL|TAR|HTA|ELF|DOCX|DOCM|LINK|XLSX|XLSM|XLSB|PPTX|PPAM|HTML)\b
```
Comment (ArikDay) [inline Packs/SecneurXAnalysis/Playbooks/Detonate_File_-_SecneurX_Analysis.yml:417]: Why not adding the "WORD" to here also? same thing will happened in this playbook.

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml [hunk 2]
```diff
@@ -372,11 +372,20 @@ tasks:
...
+          ignorecase: true
+        - operator: match
+          left:
+            value:
+              simple: inputs.File.Extension
             iscontext: true
           right:
             value:
-              simple: .*(?:EXE|DLL|PIF|CMD|BAT|COM|SCR|CPL|PDF|DOCX|DOC|DOCM|XLSX|XLS|XLSM|XLSB|PPT|PPX|PPTM|HWP|JTD|RFT|XPI|CRX|EML|MSG|CHM|JS|JSE|VBS|VBE|LNK|JAR|PS1|ZIP|7Z|RAR|ZLIB|ASPX|APS|PNG|JPEG|GIF|HTML|HTM|XHTML|SHTML)\b
+              simple: .*(?:EXE|DLL|PIF|CMD|BAT|COM|SCR|CPL|PDF|DOCX|DOC|DOCM|XLSX|XLS|XLSM|XLSB|PPT|PPX|PPTM|HWP|JTD|RFT|XPI|CRX|EML|MSG|CHM|JS|JSE|VBS|VBE|LNK|JAR|PS1|ZIP|7Z|RAR|ZLIB|ASPX|APS|PNG|JPEG|GIF|HTML|HTM|XHTML|SHTML|WORD)\b
```
Comment (ArikDay) [inline Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml:388]: No need to add it to "extension", only to "Type"

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml [hunk 3]
```diff
@@ -372,11 +372,20 @@ tasks:
       - - operator: match
           left:
             value:
-              simple: File.Type
+              simple: inputs.File.Type
+            iscontext: true
+          right:
+            value:
+              simple: .*(?:EXE|DLL|PIF|CMD|BAT|COM|SCR|CPL|PDF|DOCX|DOC|DOCM|XLSX|XLS|XLSM|XLSB|PPT|PPX|PPTM|HWP|JTD|RFT|XPI|CRX|EML|MSG|CHM|JS|JSE|VBS|VBE|LNK|JAR|PS1|ZIP|7Z|RAR|ZLIB|ASPX|APS|PNG|JPEG|GIF|HTML|HTM|XHTML|SHTML|WORD)\b
```
Comment (ArikDay) [inline Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml:379]: File type is without dot(".") please fix and do tests to make sure its working

File: Packs/JoeSecurity/ReleaseNotes/1_1_17.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Detonate File - JoeSecurity V2
+
+- Added a new extensions to supported files.
```
Comment (ArikDay) [inline Packs/JoeSecurity/ReleaseNotes/1_1_17.md:6]: added file types, not extensions

File: Packs/JoeSecurity/ReleaseNotes/1_1_17.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Detonate File - JoeSecurity V2
+
+- Added a new extensions to supported files. 
+- Updated the timeout parameter.
```
Comment (ArikDay) [inline Packs/JoeSecurity/ReleaseNotes/1_1_17.md:7]: to what and why?

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_42.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detonate File - Generic
+
+- Deleted the Interval and the Timeout input from the playbook. The playbook will use those inputs from the sub-playbook.
```
Comment (ArikDay) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_42.md:6]: You need to mention the sub playbook, in this case Joe security.

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml (hunk 3 above)
Comment (karinafishman) [reply Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml:379]: It is not a "."
. matches any character (except for line terminators)
* matches the previous token between zero and unlimited times, as many times as possible, giving back as needed

**Review by ArikDay — APPROVED** [review]

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_42.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detonate File - Generic
+
+- Deleted the Interval and the Timeout input from "Detonate File - JoeSecurity V2" playbook. The playbook will use those inputs from the sub-playbook.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_42.md:6]:
```suggestion
Deleted the *Interval* and *Timeout* inputs from **Detonate File - JoeSecurity V2** playbook. The playbook will use those inputs from the sub-playbook.
```

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml [hunk 8]
```diff
@@ -419,9 +428,9 @@ inputs:
   playbookInputQuery:
 - key: Timeout
   value:
-    simple: "1500"
+    simple: "900"
   required: false
-  description: The duration after which to stop pooling and to resume the playbook (in minutes)
+  description: The default duration after which to stop pooling and to resume the playbook (in seconds).
```
Comment (ShirleyDenkberg) [inline Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2.yml:433]: should this be pooling or polling?

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2_README.md [hunk 9]
```diff
@@ -29,7 +29,7 @@ This playbook does not use any sub-playbooks.
 | **Name** | **Description** | **Default Value** | **Required** |
 | --- | --- | --- | --- |
 | File | File object of the file to detonate. The File is taken from the context. | File | Optional |
-| Timeout | The duration after which to stop pooling and to resume the playbook \(in minutes\) | 1500 | Optional |
+| Timeout | The default duration after which to stop pooling and to resume the playbook \(in seconds\). | 900 | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2_README.md:32]: Should this be pooling or polling?

File: Packs/SecneurXAnalysis/ReleaseNotes/1_0_8.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detonate File - SecneurX Analysis
+
+- The input for supported file types task was fixed from File.Type to Inputs.File.Type.
```
Comment (ShirleyDenkberg) [inline Packs/SecneurXAnalysis/ReleaseNotes/1_0_8.md:6]:
```suggestion
Fixed the input for supported file types task from *File.Type* to *Inputs.File.Type*. 
```

**Comment by ShirleyDenkberg** [comment]
@ArikDay Doc review completed.

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2_README.md (hunk 9 above)
Comment (karinafishman) [reply Packs/JoeSecurity/Playbooks/playbook-Detonate_File_-_JoeSecurity_V2_README.md:32]: it should be polling. thanks

**Pull request merged**